doc/api/mltsp.custom_feature_tools.txt

```diff
@@ -0,0 +1,143 @@
+.. AUTO-GENERATED FILE -- DO NOT EDIT!
```

Comment:
I don't think it's that bad...the normal thing would be to write them ourselves and then they would 100% live in our repo, they'd just look a little less silly since they wouldn't say AUTO-GENERATED everywhere. For example, [scikit-learn](https://github.com/scikit-learn/scikit-learn/blob/213a24c782da9909a4078442daf13dff81bc9444/doc/modules/classes.rst) as a ton of these.